### DISCUSSION TITLE
Minishift VM startup error

### DISCUSSION THREAD

**Discussion by metin-karaca** [okd-project/okd#1946] [Q&A]
Description:
I am encountering an error while starting the Minishift cluster. I followed the installation steps as mentioned in the documentation, and everything seemed to go smoothly until I tried to start the cluster using the command minishift start. However, I'm getting a timeout error during the cluster startup process.

Steps:

Installed Minishift using the provided installation script.
Executed the command minishift start --memory 30g --cpus 12 --disk-size 100gb.
Observed the timeout error during the cluster startup process. 
Can't access  http://minishift.io/index.html

Cant't login with oc login. Because can't access https://vm-ip:8443
I can see all the servers are up and running via minishift status.
I have used virtualbox instead of kvm as well. But had the same error.

System Information:

Operating System: Ubuntu 20.04 LTS
Minishift Version: v1.34.3+4b58f89
Virtualization Environment: KVM
Additional Information:
I have tried restarting the system and re-installing Minishift, but the issue persists. Any help in resolving this would be greatly appreciated.

![Screenshot 2024-06-07 155342](https://github.com/okd-project/okd/assets/61264273/244644df-465a-40e4-bfa6-a683b7a94aa4)
![Screenshot 2024-06-07 155410](https://github.com/okd-project/okd/assets/61264273/c22200b3-169d-4906-9872-6238c052907e)
![Screenshot 2024-06-07 155454](https://github.com/okd-project/okd/assets/61264273/bdf9d991-b83a-4cb2-be45-8b9464e0f1a0)

**Comment by bdlink** [comment]
Minishift is using openshift 3.11 and is no longer maintained (the last significant changes are over 4 years old).  There is an issues page at https://github.com/minishift/minishift/issues.  Their git repository recommends using CodeReady Containers (now called Openshift Local) and sends you to the RedHat site https://console.redhat.com/openshift/create/local to proceed (you will need a (free) Red Hat developers account to access this.

All of this is outside the scope of what OKD is doing, but the last time I used CRC/Openshift Local it worked fine.  If there were a significant interest in having an OKD version of CRC we could reopen this issue.  We have supported it in the past.

We are currently trying to come up with sane advice on the easiest way to get OKD installed in the home environment.  The easiest way depends on the platform you want to use to install it and your skill set.  With the hardware you are allocating to minishift (30 Gi , 12 CPUs, 100 Gi disk) you can install a single node cluster with the current OKD 4.15 and have room to spare (but more disk would be useful/needed to run applications).
